src/Examine.Test/Examine.Test.csproj

```diff
@@ -53,23 +53,21 @@
     </Content>
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.13.1" />
     <PackageReference Include="Lucene.Net.Spatial">
       <Version>4.8.0-beta00016</Version>
     </PackageReference>
-    <PackageReference Include="Microsoft.Azure.Storage.Blob">
-      <Version>11.2.2</Version>
-    </PackageReference>
-    <PackageReference Include="Microsoft.Extensions.Hosting" Version="5.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="5.0.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.9.4" />
-    <PackageReference Include="Moq" Version="4.16.1" />
+    <PackageReference Include="Microsoft.Extensions.Hosting" Version="6.0.1" />
```

Comment:
If we multi-target, then I think we need to keep the v5 versions around for the v5 output?

Reply:
All the changed packages have multi targeting and will work on both .NET 5 and .NET 6. Example: https://www.nuget.org/packages/Microsoft.Extensions.Hosting

Therefore, I don't think it makes much sense to keep two different versions. All the packages are just for testing anyways and will therefore not affect users using Examine.